setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```

Comment:
Hmm, why is this `[bdist_wheel]` now? I used to add just `[wheel]` to projects.

/cc @dstufft